Patch for deb-pkg v1.6.1

Nightwalker-87 opened the pull request:
**stlink (1.6.1+ds-3)** 
  * Corrected install paths & changelog files
  * Corrected settings for deb pkg distribution

--> This patch addresses several points from a previous discussion and user feedback from the source project.

**Notes:**

1) Though it may not be a good practise in general, the changelog has been updated as some old entries have turned out to be wrong which have been corrected in the project's own changelog in the meanwhile. To avoid this in the future the Debian changelog should cover only package related changes, while the former will only focus on software-related changes (and will not list any distribution-related issues).
2) The Debian copyright file will no longer be updated with new contributors (there are too many and possibility of text formatting is very limited). To comply with the license terms we have added a separate listing within the project which is explicitly mentioned in the copyright file. This may be transferred to a markdown list in the future. For privacy reasons we also avoid to further disclose e-mail addresses publicly where possible.

bluca commented:
This diff looks wrong? It's not just changing debian/ content, it's merging from the upstream master branch too. Was it a merge gone wrong on your local fork?

> **Notes:**
> 
> 1. Though it may not be a good practise in general, the changelog has been updated as some old entries have turned out to be wrong which have been corrected in the project's own changelog in the meanwhile. To avoid this in the future the Debian changelog should cover only package related changes, while the former will only focus on software-related changes (and will not list any distribution-related issues).

As mentioned already multiple times, the changelog will contain anything that is deemed relevant, as per debian policy. This includes non-packaging changes. And older entries will not be changed retroactively, as they are already part of the stable distribution.

> 2. The Debian copyright file will no longer be updated with new contributors (there are too many and possibility of text formatting is very limited). To comply with the license terms we have added a separate listing within the project which is explicitly mentioned in the copyright file. This may be transferred to a markdown list in the future. For privacy reasons we also avoid to further disclose e-mail addresses publicly where possible.

As mentioned already multiple times, the preferred format of the FTP team is the current format, and thus it is not going to change, to avoid rejections on future binary-new uploads and RC bugs.

Nightwalker-87 commented:
> This diff looks wrong? It's not just changing debian/ content, it's merging from the upstream master branch too. Was it a merge gone wrong on your local fork?

No, it is a patch for the latest release on our upstream `master` branch and was meant to be exclusively for the distributed deb package. That's why it will not be pushed to the projects' `master` branch currently. I addressed the points you mentioned earlier and corrected the tool/package-descriptions to align with the source project as they were outdated.

I will no longer care about the other points cited above, to be honest and will not add anything to the /debian files anymore too.
You may accept or change or close this PR - do whatever you want.

bluca commented:
As per the debian workflow,changes outside of the Debian directory cannot be committed directly. The build will break. They have to be done as out-of-tree quilt-format patches. This is not a matter of personal preference - it's how 3.0 quilt format packages work.

Nightwalker-87 commented:
So I am closing this then...
Proposed changes will remain on the `debian` branch of the `stlink` source repo for a while.
However this branch will be deleted again, shall there be no interest in working with it.
